colossalai/shardformer/policies/t5.py

```diff
@@ -170,14 +170,6 @@ def module_policy(self):
 
         # optimization configuration
         if self.shard_config.enable_fused_normalization:
-            self.append_or_create_submodule_replacement(
-                description=SubModuleReplacementDescription(
-                    suffix="layer_norm",
-                    target_module=FusedRMSNorm,
```

Comment:
Why do u remove this?

Reply:
The replacement strategy for T5LayerFF's layer_norm is duplicated.

Reply:
The replacement strategy for T5LayerFF's layer_norm is duplicated.